ddtrace/tracer/span_test.go

```diff
@@ -70,13 +71,23 @@ func TestSpanOperationName(t *testing.T) {
 func TestSpanFinish(t *testing.T) {
 	assert := assert.New(t)
 	wait := time.Millisecond * 2
+	duration := wait
 	tracer := newTracer(withTransport(newDefaultTransport()))
 	span := tracer.newRootSpan("pylons.request", "pylons", "/")
 
+	if strings.HasPrefix(runtime.GOOS, "windows") {
+		// We can still run this test on windows, but unfortunately the sleep timer is not precise enough
+		// for this to always pass. This is due to the combination of time.Sleep() with the special now()
+		// function on windows in this test. In reality, the duration is accurate, but the sleep timer is
+		// not.
+		wait = 2 * time.Second
+		duration = wait - 2*time.Millisecond
+	}
+
 	// the finish should set finished and the duration
 	time.Sleep(wait)
 	span.Finish()
-	assert.Greater(span.Duration, int64(wait))
+	assert.Greater(span.Duration, int64(duration))
```

Comment:
Not necessary but it might be a bit cleaner to just use https://pkg.go.dev/github.com/stretchr/testify/assert#InDelta
And allow for a bit of wiggle room all the time?

Reply:
In general we don't want wiggle room. It's an unfortunate consequence of the Windows APIs that we have to overcome.